### ISSUE TITLE
Error while importing "from tesserocr import PyTessBaseAPI"

### ISSUE THREAD

**Issue by llazzaro** [sirfz/tesserocr#9]
After successfull compilation the example code in the readme fails while importing  PyTessBaseAPI

```
from tesserocr import PyTessBaseAPI
  File "build/bdist.linux-x86_64/egg/tesserocr.py", line 7, in <module>
  File "build/bdist.linux-x86_64/egg/tesserocr.py", line 6, in __bootstrap__
ImportError: /home/leonardo/.python-eggs/tesserocr-2.0.2b0-py2.7-linux-x86_64.egg-tmp/tesserocr.so: undefined symbol: _ZN9tesseract11TessBaseAPI13AnalyseLayoutEb
```

**Comment by sirfz** [comment]
First time I see this one. Did you compile using Cython? (the setup compiles the accompanied cpp file if Cython is not installed) and what tesseract version you have installed?

**Comment by enkidulan** [comment]
Same here, I've built it with using local compiled `leptonica-1.73` and `tesseract-3.04.01` on `python2.7.6`. Have cpython installed in my system (3.13.0-86-generic #131-Ubuntu SMP).


The other thing that may be important it that I have run command before compiling:

    sudo apt-get install cython tesseract-ocr libtesseract-dev libleptonica-dev

so I have other version of tesseract in my system, so it probably may affect local installation if that libraries have low level of isolation...

**Comment by llazzaro** [comment]
I fixed it with 
`sudo apt-get remove tesseract-ocr `

you have to remove tesseract with apt or change with configure something (don't know hoe to do this step)

**Comment by sirfz** [comment]
It's possible tesserocr is loading the wrong library version (if you have an older tesseract version installed via apt) so @llazzaro's suggestion makes sense. Make sure you uninstall any older version of tesseract before installing a newer one.

**Comment by enkidulan** [comment]
Thanks for your responses, they give me a hint to make it work. 

I've got the core of the problem, the thing is that `tesserocr.so` uses the first `leptonica` and `tesseract` libraries it finds so it's required to set paths to right builds of leptonica and tesseract explicitly.

In may case I succeeded with setting  `LD_LIBRARY_PATH`:

     env LD_LIBRARY_PATH=/home/.../parts/leptonica/lib/:/home/.../parts/tesseract/lib/ bin/python

I guess al least it would be great to update docs with that information about how to use tesserocr with locally built libraries of `leptonica` and `tesseract`.
